### ISSUE TITLE
Make cloudfront.d.n the default mirror for EC2 images

### ISSUE THREAD

**Issue by nbraud** [andsens/bootstrap-vz#361]
As mentioned in #338, EC2 images should probably use `cloudfront.d.n` as the default mirror.
Since the discussion in the previous bug, DSA has taken over the service, and it should be suitable as a security mirror.

The advantage is twofold:
- avoid out-of-AWS traffic, with its associated latency and cost;
- avoid the extra-latency of a HTTP request to `deb.debian.org` for each query (before getting a redirect) on Jessie and earlier.

**Comment by nbraud** [comment]
@andsens Is there currently a sane-ish way to specify different defaults based on the provider?

**Comment by andsens** [comment]
No, not really. I also think that defaults which change depending on other variables can be quite unpredictable, better to go with just showing what to set it to in the example/official manifests (maybe even add a comment in the line above)

**Comment by vorlock** [comment]
Official Jessie images are using this https://github.com/andsens/bootstrap-vz/blob/master/manifests/official/ec2/ebs-jessie-amd64-hvm.yml#L31 it's the same for testing Stretch images. This can always be overwritten if one want to use different mirror, beside Stretch is hopefully coming out very soon as we're entering deep freeze in 5 days. So over all I see no reason for making this change in the source when builds still are using cloudfront.d.n.
